contrib/adapters/ale.py

```diff
@@ -58,22 +59,24 @@ def _parse_data_line(line, columns, fps):
             del metadata["Start"]
             duration = None
             end = None
-            if "Duration" in metadata:
+            if metadata.get("Duration", "") != "":
```

Comment:
I think you can do: `if metadata.get("Duration"):` since in python, `""` is the same truthiness as `None`